src/main/java/com/google/sps/data/EventUpdater.java

```diff
@@ -36,7 +53,159 @@ public Entity getEntity() {
     return this.entity;
   }
 
-  public void updateEvent(HttpServletRequest request, GivrUser user, boolean forRegistration, EmbeddedEntity historyUpdate) throws IllegalArgumentException {
+  // Called when creating and updating Event from AddEventServlet and EditEventServlet.
+  public void updateEvent(HttpServletRequest request, GivrUser user, boolean forRegistration, EmbeddedEntity historyUpdate) throws IllegalArgumentException {    
```

Comment:
... this diff looks wrong? rebase? IIRC, i think this is from Jenny's change?

Reply:
I included this file because there were fields in here that needed to be updated as well, Jenny's change with it hasn't been merged yet though